### ISSUE TITLE
AMICI simulation fails when having initial states defined as NaN in the condition table

### ISSUE THREAD

**Issue by elbaraim** [AMICI-dev/AMICI#1396]
I provide here a minimal example of code with a PEtab model in which when I set both initial conditions to numerical values in the condition table, it simulates and returns the expected value (the initial condition of the input).

However, if I set one or both to NaN (or empty string) it fails instead of simulation without doing assignments.

The minimal example reproduces the error scenario. Executing `test.py`.
[test.zip](https://github.com/AMICI-dev/AMICI/files/5905432/test.zip)

I have latest `develop` version.

**Comment by dilpath** [comment]
Could you try the above PR with the following script (and the PEtab you supplied -- thanks for that)?

```python
import petab
from amici.petab_import import import_petab_problem
from amici.petab_objective import simulate_petab


petab_problem = petab.Problem.from_yaml("test/Pom1p_Problem.yml")
amici_model = import_petab_problem(petab_problem)
result = simulate_petab(petab_problem, amici_model)
for rdata in result['rdatas']:
    print(rdata['y'])

```

**Comment by dweindl** [comment]
Related to #1319, #1345 and #1344.

**Comment by dweindl** [comment]
Should be fixed in 0.11.13
